cves/kernel/CVE-2016-4578.yml

```diff
@@ -377,8 +398,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
```

Comment:
Capitalize the beginning of this statement. Replace "before hand" with "beforehand" and remove the word "here" since it is implied. Add punctuation.